File: Study Plans/Binary Search/Python/find-the-distance-value-between-two-arrays.py
class Solution:
    def findTheDistanceValue(self, arr1: List[int], arr2: List[int], d: int) -> int:
        
        c = 0
        
        for i in arr1:
            t = True
            for j in arr2:
                if abs(i-j) <= d:
                    t = False
                    break
            if t:
                c+=1
        
        return c

File: Study Plans/Binary Search/Java/Find-the-Distance-Value-Between-Two-Arrays.java
class Solution {
    public int findTheDistanceValue(int[] arr1, int[] arr2, int d) {
        
        int count = 0;
        
        for (int i = 0; i<arr1.length; i++) {
            boolean t = true;
            for (int j = 0; j<arr2.length; j++) {
                if (Math.abs(arr1[i]-arr2[j])<=d) {
                    t = false;
                    break;
                }
            }
            
            if (t == true) {
                count++;
            }
        }
        return count;
    }
}
